### ISSUE TITLE
Allow trackbar seeking at all times and make trackpad visible in the player when autoplay is disabled

### ISSUE THREAD

**Issue by Etim-Orb** [TeamNewPipe/NewPipe#8089]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Trackbar seeking is blocked when video buffering is happening [Image 1]. Allowing the control of the slider is favorable for reasons outlined below.

### Why do you want this feature?

There have been and are problems with videos loading. If I wanted to seek to let's say the middle of the video I would have to wait for the initial buffer to load [1] then after seeking where I want, I would have to wait the buffer from the middle of the video as well. But if I could seek to the middle of the video immediately (perhaps even before starting the stream) I could reduce the time needed to start a video, and save data and time.

I could do this by starting the video stream (press play icon if autoplay is not enabled) then while buffering I could seek where I want while the stream loads. Or I could start the stream where I want before even pressing play by either having a seek bar at the press to play screen [2], or by removing the press to play screen and having the original interface [3] as the starting interface. This means I suggest replace [2] with [3].

This would mean we can set up the properties of the stream before even attempting to start it. Not every feature has to be on this starting screen that are present in the current player (such as share button etc.) but the skeleton can be the same.

I don't know how this is implemented in the vanilla YT app but this should be similar to that.

### Additional information

Image 1
![image5](https://user-images.githubusercontent.com/94188674/159231119-82b800f5-d622-42e1-a41c-051e10f3dcb4.png)
Image 2
![image6](https://user-images.githubusercontent.com/94188674/159231155-27000989-52eb-41cc-8ca7-fea1e3efa228.png)
Image 3
![image7](https://user-images.githubusercontent.com/94188674/159231174-5ba5d084-9b2d-4e30-aac5-5c5d629555ae.png)

**Comment by Etim-Orb** [comment]
@litetex This is not primarily for ATV.

**Comment by opusforlife2** [comment]
The lack of beginning and ending timestamps in the 1st screenshot seems to imply that the player doesn't even know the video length at that point, so seeking wouldn't be possible.

**Comment by Etim-Orb** [comment]
Yes that part needs to be solved. Since I'm not a developer I cannot weigh in, but those who know what they're doing should decide what needs to be loaded first and what after the stream starts, also what elements are needed on the start screen from image [3].

**Comment by opusforlife2** [comment]
Maybe we could have "blind seeking" of a sort, where it's simply assumed that the video is 10 minutes long. If the stream duration retrieved later is shorter than whatever timestamp the user is at, the player could just show the end screen with the Replay button.
